sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/SharingProfileGroupTypes.cs

```diff
@@ -18,6 +18,5 @@ public static class SharingProfileGroupTypes
     {
         public const string Subscriptions = "Subscriptions";
         public const string AADTenants = "AADTenants";
-        public const string Community = "Community";
```

Comment:
Is this expected?

Reply:
based on this change, https://github.com/Azure/azure-rest-api-specs/pull/17985/files#diff-2861a3a31f910fbcb3cbc0af6f313527001ff65e8360cfcc566ae280a27b8bf9L1849 , looks like it is.